modules/README.md

```diff
@@ -0,0 +1,44 @@
+# How To
+This module is used to organize AWS SSO group and EKS RBAC Syncronization
+
+
+### Example 1
+
+
+```
+module "sso_module" {
+source = "../module"
+
+eks_module = module.eks
+
+assignment = [{
+name      = "role1"
+group     = "development"
+namespace = "development"
+role      = [local.role1]
```

Comment:
Aren't we passing multiple roles here? Doesn't it make sense to have it in plural?